File: samps/contests/tc1/config/ISumitRM-3/submissions/wrong_answer/sumitWA.py
#
# File:    isumit.py
# Purpose: to sum the integers from stdin
# Author:  pc2@ecs.csus.edu or http://www.ecs.csus.edu/pc2
#
# $Id$
#

import sys

# softspace required to avoid extra space after print
# sys.stdout.softspace=0

hand = sys.stdin

sum = 0

for line in hand .readlines():
  num = int(line);
  if num > 0:
    sum += num

sum += 1000
print("The sum of the integers is %s"% sum)

# eof sumit.py $Id$

File: samps/contests/tc1/config/ISumitD1Sec-1/submissions/wrong_answer/sumitWA.pl

# Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
#
# File:    sumit.pl
# Purpose: to sum the integers from input from stdin
# Author:  pc2@ecs.csus.edu or http://www.ecs.csus.edu/pc2
#
# Tue Nov  1 17:46:37 PDT 2016
#

	$sum = 0;

	while (<>)
	{
		chomp;
		$num = 0 + $_;
		$sum += $num if $num > 0;

	}

	$sum += 1000;

	print "The sum of the integers is $sum\n";

# eof 
